hd/etc/templm/trl.txt

```diff
@@ -0,0 +1,25 @@
+%if;(wizard and (evar.m="MOD_IND_OK" or evar.m="MOD_FAM_OK"))

```

Comment:
You may want to check the code base, because I did not take care about this since the bootstrap PR :)